packages/generator-adp/src/translations/generator-adp.i18n.json

```
@@ -11,12 +11,23 @@
         "passwordLabel": "Password",
         "passwordTooltip": "Enter the password for the back-end system.",
         "applicationListLabel": "Application",
-        "applicationListTooltip": "Select the application for which you want to create an app variant."
+        "applicationListTooltip": "Select the application for which you want to create an app variant.",
+        "projectTypeLabel": "Project type",
+        "appInfoLabel": "Have in mind that synchronous views are detected for this application and controller extensions are not supported for such views. Controller extension functionality on these views will be disabled.",
+        "notSupportedAdpOverAdpLabel": "You have selected Adaptation Project as base. The selected system has SAPUI5 version lower than 1.90 and it does not support Adaptation Project as а base for a new Adaptation Project. You will be able to create such а project, but after deployment it will not work until the UI5 version of the system is updated.",
+        "isPartiallySupportedAdpOverAdpLabel": "You have selected Adaptation Project as а base. The selected system has SAPUI5 version lower than 1.96 and in order for your Adaptation Project based on Adaptation Project to work after deployment, you will need to apply SAP Note 756 SP0 on your system.",
```

```suggestion
        "isPartiallySupportedAdpOverAdpLabel": "You have selected 'Adaptation Project' as the base. The selected system has a SAPUI5 version lower than 1.96 and in order for your adaptation project based on adaptation project to work after deployment, you will need to apply SAP Note 756 SP0 on your system.",
```
adaptation project based on adaptation project seems a bit strange. Is this correct? If so, could you provide more details about this so I could potentially rephrase?

---

@nikmace, could you clarify this?

---

First, you can adapt a base application by creating a new adaptation project and then deploy it. After, you can create another adaptation project based on the deployed one. In the end the last project is what we call adp over adp.

---

Thanks for the explanation. Adapted the suggestion slightly.